docs/backup-restore.md

```diff
@@ -0,0 +1,109 @@
+* `scheduledBakcup.user` and `scheduledBackup.password` must be set to the correct user which has the permission to read the database to be backuped.
+
+Then, create a new cluster with the scheduled full backup configured by `helm install`, or enabling scheduled full backup for existing cluster by `helm upgrade`:
+
+```shell
+$ helm upgrade ${RELEASE_NAME} charts/tidb-cluster -f charts/tidb-cluster/values.yaml
+```
+
+### Ad-Hoc full backup
+
+Ad-hoc backup runs to complete for once. This functionality is encapsulated in another helm chart, `charts/tidb-backup`. According to the `mode` in `charts/tidb-backup/values.yaml`, this chart can perform either full backup or restore. We will cover restore operation in the [restore section](#restore) of this document. 
+
+To create an ad-hoc full backup job, modify the `charts/tidb-backup/values.yaml`:
+
+* Set the `clusterName` to the target TiDB cluster name
+* Set `mode` to `backup`
+* Set `storage.className` to the PV storage class name used for backup data
+* Adjust the `storage.size` according to your database size
+
+> **Note:** You must set the ad-hoc full backup PV's [reclaim policy](https://kubernetes.io/docs/tasks/administer-cluster/change-pv-reclaim-policy) to `Retain` to keep your backup data safe.
+
+Create a secret containing the user and password that has the permission to backup the database:
+
+```shell
+$ kubectl create secret generic backup-secret -n ${namespace} --from-literal=user=<user> --from-literal=password=<password>
+```
+
+Then run the following command to create an ad-hoc backup job:
+
+```shell
+$ helm install charts/tidb-backup --name=<backup-name> --namespace=${namespace}
+```
+
+### View backups
+
+For backups stored in PV, you can view the PVs by the following command:
+
+```shell
+$ kubectl get pvc -n ${namespace} -l app.kubernetes.io/component=backup,pingcap.com/backup-cluster-name=${cluster_name}
+```
+
+If you store your backup data to [Google Cloud Storage](https://cloud.google.com/storage/) or [Ceph Object Storage](https://ceph.com/ceph-storage/object-storage/), you may view the backups by the related GUI or CLI tool.
+
+## Restore
+
+The helm chart `charts/tidb-backup` helps restoring a TiDB cluster using backup data. To perform a restore operation, modify the `charts/tidb-backup/values.yaml`:
+
+* Set the `clusterName` to the target TiDB cluster name
+* Set the `mode` to `restore`
+* Set the `name` to the backup name you want to restore([view backups](#view-backups) helps you view all the backups available). If the backup is stored in `Google Cloud Storage` or `Ceph Object Storage`, you have to configure the corresponding section too(likely, you will continue to use the same configuration you set in the [adhoc full backup](#ad-hoc-full-backup)).
+
+Create a secret containing the user and password that has the permission to restore the database (skip this if you've already created one in the [adhoc full backup](#ad-hoc-full-backup) section):
```

Comment:
- ditto
- Try to avoid using contractions like "you've" in tech documentation. Use "you have" instead.

Reply:
got it